cmd/template/cluster/provider/capa.go

```diff
@@ -120,6 +123,45 @@ func templateClusterAWS(ctx context.Context, k8sClient k8sclient.Interface, outp
 			},
 		}
 
+		if config.AWS.ClusterType == "proxy-private" {
```

Comment:
which other values than `proxy-private` can we expect here? If this is the only one, maybe consider making this a bool flag instead. Otherwise maybe consider making constants for `proxy-private` and other valid values and failing if something else is set.

Reply:
there will be private as well. but at the moment there are two possible values.